config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -445,7 +445,7 @@ periodics:
     preset-azure-cred-only: "true"
     preset-azure-anonymous-pull: "true" # Sets REGISTRY which is needed when building CCM/CNM images
     preset-capz-windows-common: "true"
-    preset-capz-containerd-1-7-latest: "true"
+    preset-capz-windows-2022: "true"
```

Comment:
Was this change intended?

Reply:
yea, i updated the env vars so we'd use the **nightly** containerd package instead of the 1.7.x ones

Reply:
see it now, thanks!